### PR TITLE
feature/add unique constraint to user login

### PR DESCRIPTION
**Pull request by MariyaSin**
created migration unique user login

### CODE REVIEW

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20201031013800_Unique_constraint_user_login.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            migrationBuilder.AlterColumn<string>(
+                name: "Login",
+                table: "UserCredentials",
+                maxLength: 16);
+
+            migrationBuilder.AddUniqueConstraint(
+                name: "UX_login_unique",
+                table: "UserCredentials",
+                column: "Login");
+        }
```
Comment (KostinP) [inline src/UserService.Data.Provider.MsSql.Ef/Migrations/20201031013800_Unique_constraint_user_login.cs:22]: The old Constraint had no migration?

File: src/UserService.Models.Db/DbUserCredentials.cs [hunk 2]
```diff
@@ -11,13 +11,13 @@ public class DbUserCredentials
         public Guid Id { get; set; }
         [Required]
         public Guid UserId { get; set; }
-        public virtual DbUser User { get; set; }
         [Required]
         public string Login { get; set; }
         [Required]
         public string PasswordHash { get; set; }
         [Required]
```
Comment (djino97) [inline src/UserService.Models.Db/DbUserCredentials.cs:18]: this need remove and write through fluentAPI

File: src/UserService.Models.Db/DbUserCredentials.cs [hunk 3]
```diff
@@ -11,13 +11,13 @@ public class DbUserCredentials
         public Guid Id { get; set; }
         [Required]
         public Guid UserId { get; set; }
-        public virtual DbUser User { get; set; }
         [Required]
         public string Login { get; set; }
         [Required]
```
Comment (djino97) [inline src/UserService.Models.Db/DbUserCredentials.cs:16]: this need remove and write through fluentAPI

File: src/UserService.Models.Db/DbUserCredentials.cs [hunk 4]
```diff
@@ -11,13 +11,13 @@ public class DbUserCredentials
         public Guid Id { get; set; }
         [Required]
         public Guid UserId { get; set; }
-        public virtual DbUser User { get; set; }
         [Required]
```
Comment (djino97) [inline src/UserService.Models.Db/DbUserCredentials.cs:14]: this need remove and write through fluentAPI

File: src/UserService.Models.Db/DbUserCredentials.cs [hunk 5]
```diff
@@ -11,13 +11,13 @@ public class DbUserCredentials
         public Guid Id { get; set; }
         [Required]
```
Comment (djino97) [inline src/UserService.Models.Db/DbUserCredentials.cs:12]: this need remove

**Pull request closed without merging**